test/tests/dom.jsx

```diff
@@ -817,4 +834,150 @@ describe('dom renderer cases', () => {
             ]
         });
     });
+
+    it('should render as an svg element', () => {
```

Comment:
I just ran this test and it passes even without any of your changes to `src/`. Can you write a failing test first then apply the `src/` changes? Thanks!

Reply:
Can you check again for me? 

My git HEAD is `ca96a99f3d61e1b2826cf41dbcbe0138b495ff3d`, and when I checkout the `src/` directory set to jsx-pragmatic/master (`4c957775aabb7accfc5683385044b1455f89e98b`), my test fails (`npm run karma`). 


```
TOTAL: 1 FAILED, 39 SUCCESS


1) should render as an svg element
     dom renderer cases
     Error: Expected dom domNode 'svg' to be a valid element
    at test/index.js:10069:82
    at validateDOM (test/index.js:10073:18)
    at Context.<anonymous> (test/index.js:10568:21)
```